sarif_cli/scan_tables.py

```diff
@@ -88,21 +89,24 @@ def joins_for_projects(basetables, external_info):
     # if the sarif does have versionControlProvenance
     if "repositoryUri" in b.project:
         repoUri = b.project.repositoryUri[0]
+        project_name = b.project.repositoryUri[0] + "-" + extra
         e.project_id = hash.hash_unique((repoUri+extra).encode())
     else:
         repoUri = "unknown"
-    
+
     res = pd.DataFrame(data={
         "id"                 : e.project_id,
-        "project_name"       : repoUri,
+        "project_name"       : project_name,
         "creation_date"      : pd.Timestamp(0.0, unit='s'), # TODO: external info 
         "repo_url"           : repoUri, 
         "primary_language"   : b.project['semmle.sourceLanguage'][0],
-        "languages_analyzed" : ",".join(list(b.project['semmle.sourceLanguage']))
+        "languages_analyzed" : ",".join(list(b.project['semmle.sourceLanguage'])),
+        "automationDetails"  : extra,
```

Comment:
wait, should this also be commented out?

Reply:
Why?

Reply:
Never mind.

Reply:
Looks like it's used by
```
File: sarif_cli/table_joins_CLI.py
335:47:        .merge(sf(1111), how="left", left_on='automationDetails', right_on='struct_id', validate="1:m")
```
so I'm hesitant.